components/HorizontalMenu/HorizontalMenu.css.ts

```diff
@@ -17,3 +17,18 @@ export const horizontalMenuButton = style([
     color: 'primary',
   }),
 ])
+
+export const horizontalMenuWrapper = style([
```

Comment:
Thanks for the PR!

All looks great. There's just a little bit of style optimization that can be done in a few places where atoms are defined for styles, eg. : 

```ts
style([
  {
    '@media': {
      [media.min576]: {
        msOverflowStyle: 'none',
        scrollbarWidth: 'none',
        '::-webkit-scrollbar': {
          display: 'none',
        },
      },
    },
  },
atoms({
    overflowX: 'scroll',
})
])
```